core/src/test/scala/integration/kafka/admin/BrokerApiVersionsCommandTest.scala

```diff
@@ -56,21 +66,30 @@ class BrokerApiVersionsCommandTest extends KafkaServerTestHarness {
     assertTrue(lineIter.hasNext)
     assertEquals(s"${bootstrapServers()} (id: 0 rack: null) -> (", lineIter.next())
     val nodeApiVersions = NodeApiVersions.create
```

Comment:
We should change how we construct `NodeApiVersions` here,  such as `val nodeApiVersions = new NodeApiVersions(clientApis.map(ApiVersionsResponse.toApiVersion).asJava, Collections.emptyList(), false)`